### PR TITLE
Update documentation due to TSK

### PR DESCRIPTION
**Pull request by elektrischermoench**
This pull request updates TSK usage in secure storage documentation.

### CODE REVIEW

File: documentation/secure_storage.md [hunk 1]
```diff
@@ -159,6 +159,18 @@ to encrypt/decrypt the FEK.
 TSK is derived by:
 > TSK = HMAC<sub>SHA256</sub> (SSK, TA_UUID)
 
+#### TA storage space isolation
+
+OP-TEE provides different folders for different TAs in Linux file system for
+storing their own TEE files, but OP-TEE cannot prevent an attacker from
+directly copying a TEE file from one TA's folder to another TA's folder in
+Linux file system. TEE OS should have the ability to detect those kind of
```
Comment (jbech-linaro) [inline documentation/secure_storage.md]: With TSK added I believe the part here saying `"TEE OS should have the .... meet the requirement"` should be removed also?

**Comment by jbech-linaro** [comment]
Just a single comment, other than that:
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

File: documentation/secure_storage.md (hunk 1 above)
Comment (elektrischermoench) [inline documentation/secure_storage.md]: Thanks for your quick response i've added a commit which removes the part.

**Comment by jforissier** [comment]
Hi @elektrischermoench,

Can you please:
- squash the commits into a single one
- add Joakim's Reviewed-by: tag
- rebase on top of current master (so that this can be merged as a fast-forward)?

Thanks.

**Comment by elektrischermoench** [comment]
So I've squashed the commits to a single one and added the tag.

Rebasing master gives the following output:

`$ git rebase master
Current branch master is up to date.
`

**Comment by jforissier** [comment]
That would be `git rebase origin/master` (after `git fetch origin master`) since master is your current working branch. Alternatively, you could do `git pull --rebase`.

Also it is recommended to checkout a topic branch (`git checkout -b fix-doc-tsk` for instance) rather than work on master, otherwise you can't work on two topics at the same time ;)

Anyway, thanks for the update, I have rebased and merged the commit.

**Pull request closed without merging**